modules/apps/object/object-api/src/main/java/com/liferay/object/constants/ObjectEntryFolderConstants.java

```diff
@@ -19,4 +19,7 @@ public class ObjectEntryFolderConstants {
 
 	public static final long PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT = 0;
 
+	public static final String RESOURCE_NAME_OBJECT_ENTRY_FOLDER =
+		"com.liferay.object.model.ObjectEntryFolder";
+
```

Comment:
I think we can add this variable in the existing class `ObjectConstants`:
```
public class ObjectConstants {

	public static final String RESOURCE_NAME = "com.liferay.object";

    public static final String RESOURCE_NAME_OBJECT_ENTRY_FOLDER =
		    "com.liferay.object.model.ObjectEntryFolder";

}
```

See `CommerceConstants` class. 

Do you agree?